app/javascript/components/volunteer/Dashboard.jsx

```diff
@@ -34,7 +36,14 @@ class Dashboard extends React.Component {
         <h1>Volunteer Dashboard</h1>
         <p> Hello. You are an volunteer, and your email is {this.props.user.email}. </p>
 
-
+        <SideBar id={this.props.user.id}/>
+        <br/>
+        <br/>
+        <br/>
+        <br/>
+        <br/>
+        <br/>
+        <br/>
```

Comment:
i feel like this should be styled instead of using `<br />` to make space? @steveshi7

Reply:
Can I leave this here until we delete the old components for good and finalize the home page?